### DISCUSSION TITLE
what is individual risk measure rk?

### DISCUSSION THREAD

**Discussion by leodecarlo** [mwelch750/SDCPractice#47] [General]
Hi,

I am reading the page here https://sdcpractice.readthedocs.io/en/latest/measure_risk.html#categorical-key-variables-and-frequency-counts .

In table 1, what is and how it is computed rk ? (I mean how I should get from the dataset with expliciti computations?)

print(head(sdcInitial@origData))

Residence Gender Educ Lstat Health Weights risk fk Fk
1 2 1 5 1 2 180                                           0.00542452 2 360
2 2 1 5 1 2 180                                           0.00542452 2 360
3 2 1 3 2 2 215                                           0.02509644 1 215
4 2 2 4 1 2 76                                             0.01256342 2 152
5 1 1 4 3 2 186                                           0.02824728 1 186
6 2 2 4 1 1 76                                             0.01256342 2 152

I don't get it from the tutorial up that point.

thanks
Leonardo

**Comment by thijsbenschop** [comment]
Hi,

besides the reference that was shared here: https://github.com/sdcTools/sdcMicro/issues/350, it may be informative to go through the function descriptions in the R code in the sdcMicro package. This was helpful for me to exactly understand how all measures and methods are implemented. Some part is written in C++, but it's fairly straightforward to follow. The actual implementation may be more informative than the mathematical formula.

**Comment by leodecarlo** [comment]
Thanks,

I suppose you mean the github for development.

Leonardo
